lib/widgets/auth/auth_page/auth_page.dart

```diff
@@ -39,11 +39,11 @@ class _AuthPageState extends State<AuthPage> {
                 case AuthStep.signingInWithGoogle:
                   return WaitingIndicator('Contacting Google...');
                 case AuthStep.signingInWithEmail:
-                  return WaitingIndicator('Signing In With Email...');
+                  return WaitingIndicator('Signing In with Email...');
                 case AuthStep.signingUpWithEmail:
-                  return WaitingIndicator('Signing Up With Email...');
+                  return WaitingIndicator('Creating your Account...');
```

Comment:
ooh i feel special now (as a user)

Reply:
Haha YOUR account... I actually had it as Creating an Account but had a last minute thought :-)

Reply:
noice haha